pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -62,58 +62,88 @@ func newGitHubScraper(
 	}
 }
 
-func (ghs *githubScraper) getPullRequests(
+func getTotalPrPages(
```

Comment:
This function doesn't just get the number of pull request pages. 

It also gets the number of open and closed pull requests while recording the metric for open pull requests.

Reply:
What are your thoughts on the renaming of these functions?